pkg/arm/v6/data/master-startup.sh

```diff
@@ -99,6 +99,9 @@ logger -t master-startup.sh "restore done"
 
 tuned-adm profile openshift-control-plane
 
+# we also need openshift.local.volumes dir created before xfs quota code runs
+mkdir -m 0750 /var/lib/origin/openshift.local.volumes
```

Comment:
can you change this to make the creation idempotent everywhere? 

```suggestion
mkdir -m 0750 -p /var/lib/origin/openshift.local.volumes
```

Reply:
sure!